docker/full-tls-mutinynet/docker-compose.yaml

```diff
@@ -44,6 +43,7 @@ services:
       - FM_P2P_URL=fedimint://fedimint.my-super-host.com:8173
       - FM_BIND_API=0.0.0.0:80
       - FM_API_URL=wss://fedimintd.fedimint.my-super-host.com
+      - FM_EXTRA_DKG_META=chat_server_domain=xmpp.fedimint.my-super-host.com
```

Comment:
The chat server meta field needs to be standardized by adding a description in [`docs/meta_fields`](https://github.com/fedimint/fedimint/tree/master/docs/meta_fields) or a similar external repository (in that case please prefix with the organization name).